data.txt

```diff
@@ -1,2 +1,9 @@
 https://lh3.googleusercontent.com/-7_RBa3jKDbg/Wj3Bvt_xv_I/AAAAAAAAOPI/wyg8liWiBYoOY_RdhcEppGxS_aSwMdSaACL0BGAYYCw/h786/hill-silh.gif
 https://raw.githubusercontent.com/spacerest/see_through/master/post.png
+https://media1.tenor.com/images/382a82c05537bb7607425fc0102bd28e/tenor.gif
+https://2.bp.blogspot.com/-kPmMbYCBXeA/WKyWwl0vIVI/AAAAAAABukc/nvEOHbLATxcMedk7arzI_oQfQnClO2sowCLcB/s1600/01-funny-gif-243-dog-steals-food-from-other-dog.gif
```

Comment:
oh interesting! does this display as a GIF or does it just take the first image?